framework/components/ecl-tabs/tabs.js

```diff
@@ -0,0 +1,136 @@
+// Heavily inspired by the tab component from https://github.com/frend/frend.co
+
+// Query helper
+const q = (el, ctx = document) => [].slice.call(ctx.querySelectorAll(el));
```

Comment:
I've also thought about it. Making a small set of utilities provided with `ecl-base`. Will do it :)